res/values/strings.xml

```diff
@@ -205,7 +205,7 @@
     <string name="lockscreen_access_pattern_cleared">Pattern cleared</string>
     <string name="lockscreen_access_pattern_cell_added">Pattern cell added</string>
     <string name="lockscreen_access_pattern_detected">Pattern detected</string>
-    <string name="lockscreen_access_pattern_passwd_forget">Forget password</string>
+    <string name="lockscreen_access_pattern_passwd_forgot">Forgot password</string>
```

Comment:
If we don't change the "name" attribute of this string, then other languages don't need to be changed, right?